frontend/src/js-admin/components/Production.tsx

```diff
@@ -27,7 +27,8 @@ var PERFORMERS = [{value: 'Teekkarispeksi', name: 'Teekkarispeksi'}, {value: 'N
 var DEFAULT_VALUES: IProduction = {
   performer: 'Teekkarispeksi',
   opens: '1900-01-01T00:00',
-  active: false, /*not checked*/
+  active: false,
```

Comment:
Since this is not related, prefer to keep PR's about a single chance. Not really that important here, but it's a good practice overall. No need to revert these now though.